doc/ProvisionReleaseBuild.md

```diff
@@ -1,28 +1,38 @@
-# Provision Release Build
+# Provision & Deploy Cortx Build Stack
 
 You will need to complete this [guide](https://github.com/Seagate/cortx/blob/main/doc/Release_Build_Creation.rst) before moving onto the steps below.
 
-### 1.  Change Hostname
+### Pre-requisite
+
+- Change Hostname by running,
    ```
    sudo hostnamectl set-hostname deploy-test.cortx.com
```

Comment:
May be we should say this is example only. If user already have VM with this hostname in network it will cause issue.

Reply:
OK, done